Euroleague-Graduate/Euroleague-Graduate/View/TurkishLeaguePOTableViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  TurkishLeaguePOTableViewController.swift
+//  Euroleague-Graduate
+//
+//  Created by Modeso on 7/24/17.
+//  Copyright © 2017 Modeso. All rights reserved.
+//
+
+import UIKit
+import XLPagerTabStrip
+
+class TurkishLeaguePOTableViewController: UITableViewController, IndicatorInfoProvider {
+
+    private var schedule: [Array<GameData>]?
+    private let round = "PO"
+    
+    private var turkishViewModel: TurkishLeagueViewModel? = nil
+    
+    init(style: UITableViewStyle, viewModel: TurkishLeagueViewModel) {
+        turkishViewModel = viewModel
+        super.init(style: style)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    //    turkishViewModel = TurkishLeagueViewModel()
+        schedule = turkishViewModel?.getDataOfRound(round)
+        tableView.register(UINib(nibName: "TurkishLeagueTableCell", bundle: Bundle.main), forCellReuseIdentifier: "TurkishLeagueCell")
+        //        tableView.register(TurkishLeagueTableViewCell.self, forCellReuseIdentifier: "TurkishLeagueCell")
+        tableView.backgroundColor = UIColor.darkGray
+        tableView.estimatedRowHeight = 150
+        tableView.rowHeight = 150
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        tableView.reloadData()
+    }
+    
+    func indicatorInfo(for pagerTabStripController: PagerTabStripViewController) -> IndicatorInfo {
+        return IndicatorInfo(title: round)
+    }
+    
+    // MARK: - Table view data source
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return schedule?.count ?? 1
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return schedule?[section].count ?? 1
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "TurkishLeagueCell", for: indexPath)
+        let game = schedule?[indexPath.section][indexPath.row]
+        if let leagueCell = cell as? TurkishLeagueTableViewCell {
+            leagueCell.game = game
+        }
+        return cell
+    }
+    
+    func convertDateToString(_ date: Date) -> String {
```

Comment:
convertDateToString
u r trying to perform oepration on Date... 

better u can do extension for the Date and add toString method... 
i.e

extension Date {
func toString() -> String..  {
}
}

Please read this two articles for how to group ur methods using extensions..
https://www.natashatherobot.com/using-swift-extensions/
https://www.appcoda.com/protocols-in-swift/